CorpusSearch/ClientApp/src/routes/BitPlayer.tsx

```diff
@@ -63,23 +63,72 @@ function Video({vId="rLEBp8R1_XA"}) {
         setKtime(r_player.current.getCurrentTime())
         restart()
     }
-
     /* eslint-enable */
+    const alterTime = (timeString : string,timeFunc :React.Dispatch<React.SetStateAction<string>> ,chg : number) => {
+        const time_val = parseFloat(timeString) + chg
+        timeFunc(time_val.toFixed(2))
+    }
+
+    const alterStartTime = (chg : number) => {
+        alterTime(startTime,setStartTime,chg)
+    }
+    const alterEndTime = (chg : number) => {
+        alterTime(endTime, setEndTime, chg)
+    }
+    const fbwdStart = () => {
+        alterStartTime(-1)
+    }
+    const bwdStart = () => {
+        alterStartTime(-.1)
+    }
+    const fbwdEnd = () => {
+        alterEndTime(-1)
+    }
+    const bwdEnd = () => {
+        alterEndTime(-.1)
+    }
+    const ffwdStart = () => {
+        alterStartTime(1)
+    }
+    const fwdStart = () => {
+        alterStartTime(.1)
+    }
+    const ffwdEnd = () => {
+        alterEndTime(1)
+    }
+    const fwdEnd = () => {
+        alterEndTime(.1)
+    }
```

Comment:
More readable to inline these in the JSX:

```tsx
<button onClick={() => alterStartTime(-1)}>⏮️</button>
```

Reply:
Done.